Library/Homebrew/livecheck.rb

```diff
@@ -87,6 +88,23 @@ def regex(pattern = T.unsafe(nil))
     end
   end
 
+  # Sets the `@throttle` instance variable to the provided `Integer` or returns
+  # the `@throttle` instance variable when no argument is provided.
+  sig {
+    params(
+      # Throttle rate of version patch number to use for bumpable versions.
+      rate: T.nilable(Integer),
+    ).returns(T.nilable(Integer))
+  }
+  def throttle(rate = T.unsafe(nil))
```

Comment:
Would be nicer to avoid `unsafe` here if possible.

Reply:
> Whichever we choose should be consistent with Livecheck code so can discuss in follow up.

Agreed.

Generally I favour (in descending order of preference):
- non-`nilable` types
- `nilable` types
- use of `unsafe`
- use of `must`